src/react-components/ui-root.js

```diff
@@ -169,6 +174,16 @@ class UIRoot extends Component {
   };
 
   state = {
+    hasAngryActive: false,
+    hasHappyActive: false,
+    hasSadActive: false,
+    hasSurpriseActive: false,
+    hasDisgustActive: false,
+    hasEwwActive: false,
+    hasSmileActive: false,
+    hasHeartsActive: false,
+    hasCleanActive: false,
+
```

Comment:
Instead of having a boolean state for each emoji, I'd suggest just having a single `currentEmoji` state variable and setting it to "sad", "happy", etc. or null if the emoji is cleared.